CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 * [ENHANCEMENT] Alerts: Add `MimirStoreGatewayNoSyncedTenants` alert that fires when there is a store-gateway owning no tenants. #1882
 * [BUGFIX] Fix `container_memory_usage_bytes:sum` recording rule #1865
 * [BUGFIX] Fix `MimirGossipMembersMismatch` alerts if Mimir alertmanager is activated #1870
+* [BUGFIX] Fix `MimirRulerMissedEvaluations` showing % of missed alerts as a value between 0 and 1 instead of 0 and 100. #1895
```

Comment:
I personally read it the other way. I would clarify it.

```suggestion
* [BUGFIX] Fix `MimirRulerMissedEvaluations` to show % of missed alerts as a value between 0 and 100 instead of 0 and 1. #1895
```